src/directLine.ts

```diff
@@ -12,34 +12,46 @@ interface ActivityGroup {
     watermark: string
 }
 
+interface WebSocketResponse {
+    type: string;
+    message: ActivityGroup; 
+}
```

Comment:
let's rename message -> activityGroup (this is the naming convention used elsewhere in this project)